udata/tests/plugin.py

```diff
@@ -61,13 +62,45 @@ def app(request):
     test_settings = get_settings(request)
     app = create_app(settings.Defaults, override=test_settings)
     app.test_client_class = TestClient
+    return app
+
+
+@pytest.fixture(autouse=True)
+def _load_frontend(request):
+    '''
+    Use `pytest.mark.frontend` specify that frontend/api should be loaded
+    Pass an optionnal list of modules as parameter to restrict loaded modules.
+
+    Handle backward compatibility with Class.modules attribute too
+
+    Properly unload themes when enabled.
+    '''
+    if 'app' not in request.fixturenames:
+        return
+
+    app = request.getfixturevalue('app')
+    marker = request.node.get_marker('frontend')
+    modules = set()
+
+    if marker and marker.args:
+        modules |= set(marker.args[0])
```

Comment:
no need for `|=` here? `modules` will always be `set()`.

Reply:
True. Incoming one liner